.eslintrc.js

```diff
@@ -13,5 +13,22 @@ module.exports = {
                 lintFile: './tslint.json', // path to tslint.json of your project
             },
         ],
+        'dcr/only-import-below': [
+            'warn',            
+            {
+                allowedImports: [
```

Comment:
Please can we also consider adding @guardian/src-foundations (new design system tokens to eventually replace pasteup)?